packages/app-headless-cms/src/admin/components/ContentEntries/Filters/Filters.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Form, FormOnSubmit } from "@webiny/form";
+import { useContentEntryListConfig } from "~/admin/config/contentEntries";
+import { useContentEntriesList } from "~/admin/views/contentEntries/hooks";
+
+import { FiltersContainer } from "./styles";
+
+export const Filters = () => {
+    const { browser } = useContentEntryListConfig();
+    const list = useContentEntriesList();
+
+    if (!list.showingFilters || !browser.filters.length) {
+        return null;
+    }
+
+    const applyFilters: FormOnSubmit = data => {
```

Comment:
You can wrap this in `useCallback`. Just move it above the if, because it must be executed each render.